core/server/master/src/main/java/alluxio/master/FaultTolerantAlluxioMasterProcess.java

```diff
@@ -239,4 +254,33 @@ public boolean waitForReady(int timeoutMs) {
       return false;
     }
   }
+
+  @Override
+  protected void startCommonServices() {
+    boolean isStandby = mLeaderSelector.getState() == State.STANDBY;
+    if (ServerConfiguration.getBoolean(
+        PropertyKey.STANDBY_MASTER_METRICS_SINK_ENABLED) == isStandby) {
```

Comment:
`isStandby==false` means you are primary, then no matter what `PropertyKey.STANDBY_MASTER_METRICS_SINK_ENABLED` is, you should start the metrics/webserver?